addons/account_edi_proxy_client/models/account_edi_proxy_user.py

```diff
@@ -47,13 +47,7 @@ class AccountEdiProxyClientUser(models.Model):
     private_key = fields.Binary(required=True, attachment=False, groups="base.group_system", help="The key to encrypt all the user's data")
     private_key_filename = fields.Char(compute='_compute_private_key_filename')
     refresh_token = fields.Char(groups="base.group_system")
-    proxy_type = fields.Selection(
-        selection=[],
-        string="Proxy type",
-        compute='_compute_proxy_type',
-        readonly=False,
-        store=True,
-    )
+    proxy_type = fields.Selection(selection=[], string="Proxy type", readonly=False, required=True, ondelete={})
```

Comment:
- remove `_compute_proxy_type` as well then
- `readonly` is no longer necessary
- `string` is useless
- What is the purpose of an empty `ondelete` ?